jekyll/_cci2/enable-checks.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: classic-docs
+title: "Enabling GitHub Checks"
+short-title: "Enabling GitHub Checks"
+description: "How to enable GitHub Checks for CircleCI"
+categories: [getting-started]
+order: 1
+---
+
+This document describes how to enable the GitHub Checks CircleCI Setting and authorize the CircleCI Checks app to report workflow status to the GitHub app.
+
+## Overview
+
+GitHub Checks provides your workflow status messages on the GitHub Checks page and enables you to rerun a workflow from the GitHub Checks page. 
+
+After checks are enabled, CircleCI workflow status is reported under the checks tab on GitHub and job status will no longer be displayed.
```

Comment:
That is not accurate. Job status will no longer be displayed on the pull request that was getting updated via GitHub's status API. Job status be displayed on the Checks page once a workflow is selected.